actionview/test/template/url_helper_test.rb

```diff
@@ -358,7 +358,7 @@ def test_link_tag_using_block_in_erb
   def test_link_tag_with_html_safe_string
     assert_dom_equal(
       %{<a href="/article/Gerd_M%C3%BCller">Gerd Müller</a>},
-      link_to("Gerd Müller", article_path("Gerd_Müller".html_safe))
+      link_to("Gerd Müller", article_path("Gerd_Müller"))
```

Comment:
@rafaelfranca I was about to update this as well before the merge. I am not sure what this test is supposed to verify though. Its a false positive.